test/SILGen/partial_apply_debuginfo.swift

```diff
@@ -0,0 +1,7 @@
+// RUN: %target-swift-frontend -emit-silgen %S/Inputs/partial_apply_debuginfo_other.swift -primary-file %s -module-name partial_apply_debuginfo -g -Xllvm -sil-print-debuginfo | %FileCheck %s
```

Comment:
The test should ideally be positive. Is there a visible difference in a `loc` attachment on either a debug_value or an alloc_stack that we could check for?

Reply:
I didn't see one, in fact this test passes without my change, which is why I asked :) I'm not going to merge this in its current state.

Reply:
Do you perhaps see a difference with -verbose-sil? This also prints the non-debug-info relevant details of SILLocation.